packages/manager/modules/pci/src/projects/project/workflow/add/schedule/cron-validator.js

```diff
@@ -0,0 +1,159 @@
+import map from 'lodash/map';
+
+export default class CronValidator {
+  constructor() {
+    this.monthAlias = {
+      jan: '1',
+      feb: '2',
+      mar: '3',
+      apr: '4',
+      may: '5',
+      jun: '6',
+      jul: '7',
+      aug: '8',
+      sep: '9',
+      oct: '10',
+      nov: '11',
+      dec: '12',
+    };
+
+    this.weekdaysAlias = {
+      sun: '0',
+      mon: '1',
+      tue: '2',
+      wed: '3',
+      thu: '4',
+      fri: '5',
+      sat: '6',
+    };
```

Comment:
Maybe those could be handled by `moment` ?

Reply:
I have tried it but moment returns current date, month number if wrong data/month is provided. For example it returns 7 for moment().month('abc').format('M'). Seems like there is no way to validate it.

Reply:
For months, when I `moment().month('jan').format('M')` I get `1` so it seems to work fine :) 
For days you can use `format('e')` e.g `moment().day('sun').format('e')` returns `0`
https://momentjs.com/docs/#/parsing/string-format/

Reply:
no its not because of locale, can you please check the output of this "moment().month('test').format('M')". For me this returns current month because the month provided is wrong. If user enters a wrong month in UI, it will validate it as current month and generates wrong Cron.

Reply:
The input field can and must accept numbers, special characters, or characters. For example the value can be * or 2 or 2-5 or 2/5 or jan. Since its very complex validation to use regex, I do the validation in code and left the input a free text which can take any value.